hilti/toolchain/src/compiler/printer.cc

```diff
@@ -35,15 +35,20 @@ namespace {
 struct Printer : visitor::PreOrder {
     Printer(printer::Stream& out) : _out(out) {} // NOLINT
 
-    void printFunctionType(const type::Function& ftype, const ID& id) {
+    void printFunctionType(const type::Function& ftype, const ID& id, function::CallingConvention calling_conv) {
```

Comment:
I'm wondering now if the calling convention should really be part of the function type, rather than the function itself.

Reply:
I made #2052 for this - it turned out a bit more effort than I wanted to spend on it right now, but I left the WIP patch in that issue if we pick it up in the future